media/js/base/mozilla-monitor-button.js

```diff
@@ -20,17 +20,41 @@ Mozilla.MonitorButton = (function() {
     }
 
     var buttonURL = monitorButton.getAttribute('href');
+    // strip url to everything after `?`
+    buttonURL = buttonURL.match(/\?(.*)/)[1];
+
     var destURL = monitorButton.getAttribute('data-action') + 'metrics-flow';
 
-    fetch(destURL).then(function(resp) {
+    // collect values from monitor button
+    var utmParams = window._SearchParams.queryStringToObject(buttonURL);
```

Comment:
Nit: you're getting all types of params here, not just utms, so suggest renaming this to `params`?